src/utils/contextUtils.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ *  Copyright (c) 2021 Red Hat, Inc. and others.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v2.0
+ *  which accompanies this distribution, and is available at
+ *  https://www.eclipse.org/legal/epl-v20.html
+ *
+ *  Contributors:
+ *  Red Hat Inc. - initial API and implementation
+ *  Microsoft Corporation - Auto Closing Tags
+ */
+
+import { platform } from "os";
+import { ExtensionContext } from "vscode";
+
+/**
+ * Returns the path to the extension storage given the extension context
+ *
+ * @param context the extension context
+ * @returns the path to the extension storage
+ */
+export function getGlobalStoragePath(context: ExtensionContext): string {
+  let storagePath = context.globalStorageUri.path;
```

Comment:
Did you try `fsPath` from https://code.visualstudio.com/api/references/vscode-api#Uri . Docs seem to indicate it should handle what you want.

Reply:
Thank you, this is what I needed!